waiter/src/waiter/scheduler/composite.clj

```diff
@@ -99,9 +99,13 @@
         service-id->scheduler
         (scheduler/service-id->state service-id)))
 
-  (state [_]
-    {:aggregator (query-aggregator-state-fn)
-     :components (pc/map-vals scheduler/state scheduler-id->scheduler)})
+  (state [_ include-flags]
+    (cond-> {:supported-include-params ["aggregator" "components"]
+             :type "CompositeScheduler"}
```

Comment:
Isn't it redundant to include the word "Scheduler" in each of these scheduler types?

Reply:
Dropped the `Scheduler` portion from the types

Reply:
Nick looks like he covered this.

Reply:
I liked it how it was before.  The type name is supposed to match the actual type name, yes?

Reply:
I'm happy to revert it :)

Reply:
Reverted to include `Scheduler` suffix.

Reply:
I still disagree that it's useful if _every possible value_ of this string is _always_ going to end in "Scheduler", but this is just an aesthetic thing so I'm not going to push it. I don't anticipate ever using this field for anything other than visually confirming that I'm looking at the state that I expected.